.github/container/test-maxtext.sh

```diff
@@ -232,8 +232,7 @@ export BASE_XLA_FLAGS=${BASE_XLA_FLAGS:---xla_gpu_enable_latency_hiding_schedule
                 --xla_gpu_enable_pipelined_all_gather=true 
```

Comment:
For another PR: Those flags default to true:
la_gpu_enable_pipelined_all_gather
xla_gpu_enable_pipelined_reduce_scatter
xla_gpu_enable_pipelined_all_reduce

Reply:
sounds good!